### PR TITLE
Feature/remove paths and search

### PR DESCRIPTION
**Pull request by tati**
This PR should remove paths and search from all nav states.

Looks good locally for the admin view...any advice on how to check student view? Is there an email I can log into hawk with?

screen shot 2016-03-17 at 4 24 57 pm

### CODE REVIEW

File: src/AppBar/linkSet.es6 [hunk 1]
```diff
@@ -63,12 +51,10 @@ else {
         if (user.access.indexOf('core-student') >= 0) {
             defaults(home, config.dashboard);
             main.push(home);
-            main.push(config.paths);
         }
         else if (/tfl/.test(user.student_type)) {
             assign(home, {
                 host: config.projects.host,
-                url: config.paths.url
```
Comment (learykara) [inline src/AppBar/linkSet.es6]: where are tfl students' home buttons supposed to lead?

**Comment by learykara** [comment]
for looking at student view... why not just impersonate?

File: src/AppBar/linkSet.es6 (hunk 1 above)
Comment (tati) [inline src/AppBar/linkSet.es6]: ah good catch, chatted with Varun and he says should go to workshops so I am removing workshops from the TFL nav but making the home button go there.

**Comment by tati** [comment]
Where do you normally get the emails from to impersonate them? I am getting this error. Or is it something to do with my db...I did a sanitize backup recently 

screen shot 2016-03-17 at 5 14 08 pm

**Comment by learykara** [comment]
you need `*()*` at the end of non-admin emails locally.

**Comment by tati** [comment]
gotcha gracias...should be good to go now and I looked locally

**Comment by learykara** [comment]
might be smart to also get rid of the hacky logic in the actual nav component that adds the search link (example [here](https://github.com/Thinkful/ui/blob/master/src/AppBar/Navigation.jsx#L86))

**Comment by learykara** [comment]
:+1:

**Pull request merged**